gpustack/policies/candidate_selectors/vllm_resource_fit_selector.py

```diff
@@ -47,13 +48,24 @@ def estimate_model_vram(model: Model) -> int:
     # https://github.com/vllm-project/vllm/blob/v0.6.1/vllm/worker/model_runner.py#L1313
     cuda_graph_size = 2 * 1024**3
     weight_size = 0
+    timeout_in_seconds = 5
     try:
         if model.source == SourceEnum.HUGGING_FACE:
-            weight_size = get_hf_model_weight_size(model.huggingface_repo_id)
+            weight_size = await asyncio.wait_for(
+                asyncio.to_thread(get_hf_model_weight_size, model.huggingface_repo_id),
+                timeout=timeout_in_seconds,
+            )
         elif model.source == SourceEnum.MODEL_SCOPE:
-            weight_size = get_ms_model_weight_size(model.model_scope_model_id)
+            weight_size = await asyncio.wait_for(
```

Comment:
Should we wait_for estimate_model_vram so we don't do this twice?

Reply:
Need to return the vram even if an exception occurs, so wait_for here to catch the error, and then we can continue to return the vram.